modules/scheduler/scheduler/src/main/resources/webapp/js/directives.js

```diff
@@ -5,6 +5,79 @@
 
     var directives = angular.module('scheduler.directives', []);
 
+    /* General date time picker that can work as either date, time or datetime picker */
+    directives.directive('dateTimePicker', function ($timeout) {
```

Comment:
don't we have reusable datepickers? and wasn't this removed from teh scheduler module a few weeks ago?